### PR TITLE
releasing v1.33.0

### PR DESCRIPTION
**Pull request by dschep**
Also modified the prs since release script to correctly get the most recent tag, grab PR descriptions from github's API, & modified its out put to be able to go straight into the changelog!

And added a step to the release process to update the docs since I changed them to not point to master to avoid issues like #5479

### CODE REVIEW

**Comment by softprops** [comment]
So excited for this release thanks for making this happen!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,64 @@
+# 1.33.0 (13.15.2018)
```
Comment (horike37) [inline CHANGELOG.md]: >13.15.2018

Is this typo?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless",
-  "version": "1.32.0",
+  "version": "1.33.0",
```
Comment (pmuens) [inline package.json]: Can you also re-generate `package-lock.json`

**Comment by dschep** [comment]
@pmuens @eahefnawy @horike37, are not all `complex-integration-tests` expected to pass? Here's my results from running them for the previous release and master: https://gist.github.com/dschep/01dbd585082e9ab7c5c19897c98eab62

**Comment by dschep** [comment]
Great. Yeah, all the simple ones pass. And actually 1 more complex tests passed for me than on v1.32.0 🙂

**Pull request merged**